home/import_ordered_content_sets.py

```diff
@@ -155,7 +160,10 @@ def _create_ordered_set_from_row(
         :return: An instance of OrderedContentSet.
         :raises ImportException: If time, units, before_or_afters, page_slugs and contact_fields are not all equal length.
         """
-        ordered_set = self._get_or_init_ordered_content_set(row, row["Name"])
+        ordered_set = self._get_or_init_ordered_content_set(
+            row, row["Slug"], row["Locale"]
```

Comment:
Do we care about case here? Should we perhaps normalise to lowercase when we read from CSV/XLSX?